collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.java

```diff
@@ -40,7 +38,6 @@
     private final LaunchState launchState;
     private final AppUpgrader appUpgrader;
 
-    @Inject
```

Comment:
I got rid of this as we generally don't use constructor `@Inject`s, and I only had it as a convenience for myself while I messed around with the signature.